addon/mixins/popup.js

```diff
@@ -63,7 +63,7 @@ export default Mixin.create({
   didCreateLayer() {
     this._super(...arguments);
     if (this.get('hasBlock')) {
-      this._popup = this.L.popup({}, this._layer);
+      this._popup = this.L.popup(this.get('popupOptions'), this._layer);
       this._popup.setContent(this.get('destinationElement'));
       this._layer.bindPopup(this._popup, this.get('popupOptions'));
```

Comment:
I never got around to making a PR on this (i can), but this is the right approach: See
PR #66  

```js
      let popupOptions = this.get('popupOptions') ? this.get('popupOptions') : {};
      this._popup = this.L.popup(popupOptions, this._layer);
      this._popup.setContent(this.get('destinationElement'));
      this._layer.bindPopup(this._popup);
```

https://github.com/donaldwasserman/ember-leaflet/blob/popup_fix/addon/mixins/popup.js